common/arch/win32/rbaudio.cpp

```diff
@@ -221,9 +236,9 @@ void RBAStop()
 
     if ((mciError = mciSendCommand(wCDDeviceID, MCI_STOP, MCI_WAIT, 0)))
     {
-		Warning("RBAudio win32: could not stop music (%lx)", mciError);
+        Warning("RBAudio win32/MCI: could not stop music (%lx)", mciError);
     }
-	redbook_finished_hook = NULL;
+    redbook_finished_hook = NULL;
```

Comment:
Prefer `nullptr` over `NULL`.

Reply:
NULL is also used by the SDL code right now too, so it should be changed in both.